src/utils/AriaLive.ts

```diff
@@ -0,0 +1,61 @@
+import 'styling/accessibility/_ScreenReader.scss';
+import { $$ } from './Dom';
+import { QueryEvents, IQuerySuccessEventArgs, IQueryErrorEventArgs } from '../events/QueryEvents';
+import { QuerySummaryUtils } from './QuerySummaryUtils';
+import { l } from '../strings/Strings';
+
+export class AriaLive {
+  private ariaLiveEl: HTMLElement;
+
+  constructor(private searchInterface: HTMLElement) {
```

Comment:
I'd use `root` instead, it's a terminology that is used a lot throughout the codebase.